### PR TITLE
MAT-6106: 64 Character Limit for Library Names

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-6106](https://jira.cms.gov/browse/MAT-6106)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-java-models/pull/160
https://github.com/MeasureAuthoringTool/cql-library-service/pull/52

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/createDraftDialog/CreateDraftDialog.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ const CreatDraftDialog = ({ open, onClose, onSubmit, cqlLibrary }) => {
     } as CqlLibrary,
     validationSchema: Yup.object().shape({
       cqlLibraryName: Yup.string()
-        .max(255, "Library name cannot be more than 255 characters.")
+        .max(255, "Library name cannot be more than 64 characters.")
```
Comment (jkotanchik-SB) [inline src/components/createDraftDialog/CreateDraftDialog.tsx]: max still set to 255

File: src/validators/CqlLibrarySchemaValidator.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Model } from "@madie/madie-models";
 
 export const CqlLibrarySchemaValidator = Yup.object().shape({
   cqlLibraryName: Yup.string()
-    .max(255, "Library name cannot be more than 255 characters.")
+    .max(255, "Library name cannot be more than 64 characters.")
```
Comment (jkotanchik-SB) [inline src/validators/CqlLibrarySchemaValidator.ts]: same thing here.

File: src/components/createDraftDialog/CreateDraftDialog.tsx (hunk 1 above)
Comment (ethankaplan) [reply src/components/createDraftDialog/CreateDraftDialog.tsx]: It's on my local as 64 and it's saying it's up to date when I try to pull/push. Weird. Edited in github

**Review by RohitKandimalla — APPROVED** [review]
